src/stub/parser.rs

```diff
@@ -261,7 +261,7 @@ impl<'a> Parser<'a> {
             buf.push(token)
         }
 
-        if buf.join("").is_empty() {
+        if buf.iter().all(|s| s.is_empty()) {
```

Comment:
Sure why not :D

I guess your way is clearer if we keep in mind the `token.is_empty` of parse_variables.

Reply:
My motivation behind this was to avoid allocating an unnecessary `String` with `join`. I think this will be replaced later anyway.